kubernetes/helm/templates/client-rbac.yaml

```diff
@@ -14,6 +14,38 @@ metadata:
   name: {{ template "name-client" . }}
   labels:
     application: {{ template "name-client" . }}
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - services
+    verbs:
+      - get
+      - list
+      - watch
```

Comment:
Just to be sure, client needs a "view" of other services to be able to get it's details, ports & stuff to expose it within it's own namespace?

Reply:
Client & server have exactly the same responsibilities concerning exposing services, remeber that exposing can be bi-directional.